app/experimenter/experiments/tests/test_migrations.py

```diff
@@ -1,118 +1,42 @@
-from django.utils.text import slugify
+from django.contrib.auth.models import User
 from django_test_migrations.contrib.unittest_case import MigratorTestCase
-from faker import Factory as FakerFactory
 
-from experimenter.experiments.constants import NimbusConstants
+from experimenter.experiments.tests.factories import (
+    ExperimentChangeLogFactory,
+    ExperimentCommentFactory,
+    ExperimentFactory,
+    NimbusChangeLogFactory,
+    NimbusExperimentFactory,
+)
+from experimenter.notifications.tests.factories import NotificationFactory
+from experimenter.openidc.tests.factories import UserFactory
 
-faker = FakerFactory.create()
 
-
-def create_experiments(User, NimbusExperiment, *args):
-    owner = User.objects.create(
-        first_name="foo",
-        last_name="bar",
-        email="foo@example.com",
-        username="foobar",
-    )
-    return [
-        NimbusExperiment.objects.create(
-            name=faker.catch_phrase(),
-            slug=slugify(faker.catch_phrase())[: NimbusConstants.MAX_SLUG_LEN],
-            owner=owner,
-            **props,
-        ).id
-        for props in args
-    ]
-
-
-class TestMigration0174Forward(MigratorTestCase):
-    migrate_from = ("experiments", "0174_nimbusexperiment_results_data")
-    migrate_to = ("experiments", "0175_switch_to_status_next")
+class TestMigration0181Forward(MigratorTestCase):
```

Comment:
Probably ignorable out-loud thinking: Do we need to delete old migration tests? Could be handy to keep them around, if only as example code. Then again, migrations shouldn't ever change or have cause to fail from other future changes - so I guess that makes the tests rather stale code. (And git history is a thing) 🤷‍♂️

Reply:
No I don't think so, they only need to run once against prod and then never again, so I've always deleted them after they're used.  And yeah if you want to see them they're in the history.